### DISCUSSION TITLE
EDFA advanced model x-axis for nf_ripple, dgt, gain_ripple

### DISCUSSION THREAD

**Discussion by caffery-chen** [Telecominfraproject/oopt-gnpy#424] [General]
Dear contributor,

After searching in the issues, it is still not that clear to me which x-axis is used for nf_ripple, dgt, gain_ripple, respectively?

It seems dgt has a reversed frequency axis from 196.1 ~ 191.3.

How about nf_ripple, gain_ripple?

I am currently using the Version: 2.2.0.

Thanks.

**Comment by ojnas** [comment]
All of them use a (non-reversed) frequency axis. There was a [bug](https://github.com/Telecominfraproject/oopt-gnpy/issues/390) in that the values in some example files were specified vs. wavelength (i.e. reversed frequency) instead but that has been [fixed](https://github.com/Telecominfraproject/oopt-gnpy/commit/3a72ce84d03f500ca6cda07dcafa882ecdfeebc4).

Version 2.2 is very old and you really should update.
